relay-single.ts

```diff
@@ -381,7 +381,10 @@ export class SingleRelayConnection implements Subscriber, SubscriptionCloser, Ev
 
             ws = this.wsCreator(this.url, this.log);
             if (ws instanceof Error) {
-                console.error(ws);
+                console.error(ws.name, ws.message, ws.cause);
+                if (ws.name == "SecurityError") {
+                    return ws;
```

Comment:
This is another example of the poor legacy API & implementations of the web browser platform.